docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -480,3 +480,6 @@ Supported variable names are:
     found, or if the selected value is not a supported type, then no header is emitted. The
     namespace and key(s) are specified as a JSON array of strings. Finally, percent symbols in the
     parameters **do not** need to be escaped by doubling them.
+
+%START_TIMESTAMP%
```

Comment:
We already have START_TIME in access logs. I wonder, could we just use START_TIME and allow a custom format specifier that is effectively milliseconds since epoch? Then we could use the same code in both places. If we can't do that what about START_TIME_SINCE_EPOCH or something? Though I like the former better. Thoughts?

Reply:
@mattklein123 do you think specifically handling `since_epoch_ms` in start time formatting makes sense? 

i.e. the start time formatting is either: 
1. Using `std::put_time`'s `fmt` string for `%START_TIME(%Y/%m/%dT%H:%M:%S%z %s)%`, or
2. Using `fmt::format` for `%START_TIME(since_epoch_ms)%`

Reply:
However, I think for now we probably can go with `START_TIME_SINCE_EPOCH` for header formatter, since currently the way `AccessLogFormatParser` build the formatter is a bit isolated to itself. WDYT?

Reply:
This is somewhat related to https://github.com/envoyproxy/envoy/issues/3256. It would be nice to support custom formatters apart from what is currently supported. cc @bplotnick.

I think if we did this we could support time since epoch and some other things and I think this is a better long term direction, though more work. Are you interested in picking up #3256 as well and working on both?

Reply:
@mattklein123 sure let me take a look at the possibilities.